sdk/tests/deqp/framework/common/tcuTestCase.js

```diff
@@ -271,7 +272,6 @@ goog.scope(function() {
 
         // If no more children, get the next brother
         if (test == null && this.parentTest != null) {
-            this.currentTestNdx = 0;
```

Comment:
Thanks for the information. I didn't notice them, but was mainly testing with the transform feedback tests, and using a filter to only run a subset of the tests.